packages/db/src/index.ts

```diff
@@ -1,11 +1,264 @@
+/**
```

Comment:
Normally this might go into a README, but I want the README to remain to distinct from the API documentation.

I'm looking into options for how we can include arbitrary markdown files, but existing plugins don't seem to have compatibility with latest TypeDoc yet.

Reply:
Hm. I tried doing this via the [`includes`](https://typedoc.org/guides/options/#includes) mechanism, but it turns out that you can't do `[[include:...]]` from insight a file that's already being `[[included:...]]`-ed.

I think keeping this here for now seems like the least bad option? I could move it to the end of the file; would that be better?